wikitemplate.md

```diff
@@ -56,6 +57,7 @@
 - [ ] Verify you are able to sync two devices using the secret code
 - [ ] Visit a site on device 1 and change shield setting, ensure that the saved site preference is synced to device 2
 - [ ] Enable Browsing history sync on device 1, ensure the history is shown on device 2
+- [ ] Clear browsing history on device 1, ensure the history is sync back on device 1 from device 2
```

Comment:
Did you mean to include this?

Reply:
Yes added it because of https://github.com/brave/sync/issues/182. Not really sure if that is expected or not but since that's been the behaviour added it.